R/onload.R

```diff
@@ -28,4 +28,7 @@
     systemfonts::register_font("fallback", systemfonts::font_info("")$path[1])
   }
 
+  # Only register once sf is loaded
+  vctrs::s3_register("sf::st_as_grob", "sfc_labelled")
+  vctrs::s3_register("sf::st_as_grob", "sfc_textbox")
```

Comment:
We didn't register these methods anywhere, but somehow it worked anyway.  It is better to register them, and to do that we need to register when {sf} is loaded. We could technically avoid a dependency on {vctrs} for this, but it might have uses in the future and it is a 'free' dependency as ggplot2 also takes it as a dependency.